recipes/ampgen/LICENSE.txt

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You shouldn't need this file, conda-build will automatically look in the source tarball for matching files (you still need the licence_file field in the YAML)